### ISSUE TITLE
Unable to connect to Docker container on remote Ubuntu machine

### ISSUE THREAD

**Issue by tt4g** [microsoft/vscode-remote-release#7654]
Note Japanese was included in log messages and command output, but has been edited to English whenever possible.

After connecting to a remote Ubuntu machine from my local Windows machine via remote SSH extension,
when trying to connect to Docker container on that machine,
the error "Shell server terminated (code: 255, signal: null )" is reported (the attached log shows the whole error).

A very similar issue has been reported (#7507). However, in my case the cause was different.

The output of [the comment](https://github.com/microsoft/vscode-remote-release/issues/7507#issuecomment-1313489312) command is as follows:

```shell
$ wmic process where (commandline like "%ssh-askpass.bat%") get processid,parentprocessid /format:csv

Node,ParentProcessId,ProcessId
EDITED_DEADBEAF,10264,1848

wmic process where (commandline like "%ssh-askpass.bat%") get processid,parentprocessid /format:"%WINDIR%\System32\wbem\en-us\csv"
XSL Invalid XSL format (or) file name.
```

The file `%WINDIR%\System32\wbem\en-US\csv.xsl` was missing in local Windows machine.
But the file `%WINDIR%\System32\wbem\en-JP\csv.xsl` exists.

I tried the next extension version, but it did not solve my problem.

- Dev Containers: v0.266.1 (v0.267.0 pre-release)
- Remote - SSH: v0.92.0 (v0.93.2022112815 pre-release)

---

- VSCode Version: 1.74.0
- Local OS Version: Windows 10 (`ver` command output: `Microsoft Windows [Version 10.0.17763.3650]`)
- Remote OS Version: Ubuntu 20.04.5 LTS
- Remote Extension/Connection Type: Containers via SSH

- Logs:

    <details>
    <summary>C:\Users\EDITED_DEADBEAF\AppData\Roaming\Code\logs\20221208T124355\window3\renderer.log</summary>

    ````
    2022-12-08 12:45:33.395 [info] [attempt 1] Invoking resolveAuthority(ssh-remote)
    2022-12-08 12:45:40.608 [error] [Extension Host] (node:2936) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
    (Use `Code --trace-deprecation ...` to show where the warning was created)
    2022-12-08 12:45:40.627 [info] [attempt 1] resolveAuthority(ssh-remote) returned '127.0.0.1:54717' after 7232 ms
    2022-12-08 12:45:42.457 [warning] No search provider registered for scheme: vscode-remote, waiting
    2022-12-08 12:45:42.949 [error] Could not find pty on pty host: CodeExpectedError: Could not find pty on pty host
        at h.J (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:27:6395)
        at h.updateTitle (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:27:2665)
        at Object.call (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:17:5110)
        at r.q (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:15:5390)
        at r.p (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:15:4881)
        at /home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:15:4286
        at n.invoke (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:11:145)
        at C.deliver (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:11:2038)
        at v.fire (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:11:1676)
        at process.O (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:9:20886)
        at process.emit (node:events:526:28)
        at emit (node:internal/child_process:938:14)
        at processTicksAndRejections (node:internal/process/task_queues:84:21)
    2022-12-08 12:45:43.787 [info] [perf] Render performance baseline is 50ms
    2022-12-08 12:45:50.984 [error] [Extension Host] Shell server terminated (code: 255, signal: null)

    XSL Invalid XSL format (or) file name.

    node:child_process:925
        throw err;
        ^

    Error: Command failed: wmic process where (commandline like "%ssh-askpass.bat%") get processid,parentprocessid /format:"%WINDIR%\System32\wbem\en-us\csv"
    XSL Invalid XSL format (or) file name.



        at checkExecSyncError (node:child_process:848:11)
        at Object.execSync (node:child_process:922:15)
        at Object.<anonymous> (node:electron/js2c/asar_bundle:5:12704)
        at Object.<anonymous> (c:\Users\EDITED_DEADBEAF\.vscode\extensions\ms-vscode-remote.remote-containers-0.266.1\dist\common\sshAskpass.js:5:808)
        at Module._compile (node:internal/modules/cjs/loader:1165:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
        at Module.load (node:internal/modules/cjs/loader:1035:32)
        at Module._load (node:internal/modules/cjs/loader:876:12)
        at c._load (node:electron/js2c/asar_bundle:5:13343)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
      status: 44210,
      signal: null,
      output: [
        null,
        Buffer(0) [Uint8Array] [],
        Buffer(41) [Uint8Array] [
           88,  83,  76,  32, 130, 204, 140,  96,
          142, 174, 130, 220, 130, 189, 130, 205,
          131, 116, 131,  64, 131,  67, 131, 139,
          150, 188, 130, 170, 150, 179, 140, 248,
          130, 197, 130, 183, 129,  66,  13,  13,
           10
        ]
      ],
      pid: 8228,
      stdout: Buffer(0) [Uint8Array] [],
      stderr: Buffer(41) [Uint8Array] [
         88,  83,  76,  32, 130, 204, 140,  96,
        142, 174, 130, 220, 130, 189, 130, 205,
        131, 116, 131,  64, 131,  67, 131, 139,
        150, 188, 130, 170, 150, 179, 140, 248,
        130, 197, 130, 183, 129,  66,  13,  13,
         10
      ]
    }
    XSL Invalid XSL format (or) file name.

    node:child_process:925
        throw err;
        ^

    Error: Command failed: wmic process where (commandline like "%ssh-askpass.bat%") get processid,parentprocessid /format:"%WINDIR%\System32\wbem\en-us\csv"
    XSL Invalid XSL format (or) file name.



        at checkExecSyncError (node:child_process:848:11)
        at Object.execSync (node:child_process:922:15)
        at Object.<anonymous> (node:electron/js2c/asar_bundle:5:12704)
        at Object.<anonymous> (c:\Users\EDITED_DEADBEAF\.vscode\extensions\ms-vscode-remote.remote-containers-0.266.1\dist\common\sshAskpass.js:5:808)
        at Module._compile (node:internal/modules/cjs/loader:1165:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
        at Module.load (node:internal/modules/cjs/loader:1035:32)
        at Module._load (node:internal/modules/cjs/loader:876:12)
        at c._load (node:electron/js2c/asar_bundle:5:13343)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
      status: 44210,
      signal: null,
      output: [
        null,
        Buffer(0) [Uint8Array] [],
        Buffer(41) [Uint8Array] [
           88,  83,  76,  32, 130, 204, 140,  96,
          142, 174, 130, 220, 130, 189, 130, 205,
          131, 116, 131,  64, 131,  67, 131, 139,
          150, 188, 130, 170, 150, 179, 140, 248,
          130, 197, 130, 183, 129,  66,  13,  13,
           10
        ]
      ],
      pid: 14452,
      stdout: Buffer(0) [Uint8Array] [],
      stderr: Buffer(41) [Uint8Array] [
         88,  83,  76,  32, 130, 204, 140,  96,
        142, 174, 130, 220, 130, 189, 130, 205,
        131, 116, 131,  64, 131,  67, 131, 139,
        150, 188, 130, 170, 150, 179, 140, 248,
        130, 197, 130, 183, 129,  66,  13,  13,
         10
      ]
    }
    Permission denied, please try again.
    XSL Invalid XSL format (or) file name.

    node:child_process:925
        throw err;
        ^

    Error: Command failed: wmic process where (commandline like "%ssh-askpass.bat%") get processid,parentprocessid /format:"%WINDIR%\System32\wbem\en-us\csv"
    XSL Invalid XSL format (or) file name.



        at checkExecSyncError (node:child_process:848:11)
        at Object.execSync (node:child_process:922:15)
        at Object.<anonymous> (node:electron/js2c/asar_bundle:5:12704)
        at Object.<anonymous> (c:\Users\EDITED_DEADBEAF\.vscode\extensions\ms-vscode-remote.remote-containers-0.266.1\dist\common\sshAskpass.js:5:808)
        at Module._compile (node:internal/modules/cjs/loader:1165:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
        at Module.load (node:internal/modules/cjs/loader:1035:32)
        at Module._load (node:internal/modules/cjs/loader:876:12)
        at c._load (node:electron/js2c/asar_bundle:5:13343)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
      status: 44210,
      signal: null,
      output: [
        null,
        Buffer(0) [Uint8Array] [],
        Buffer(41) [Uint8Array] [
           88,  83,  76,  32, 130, 204, 140,  96,
          142, 174, 130, 220, 130, 189, 130, 205,
          131, 116, 131,  64, 131,  67, 131, 139,
          150, 188, 130, 170, 150, 179, 140, 248,
          130, 197, 130, 183, 129,  66,  13,  13,
           10
        ]
      ],
      pid: 10320,
      stdout: Buffer(0) [Uint8Array] [],
      stderr: Buffer(41) [Uint8Array] [
         88,  83,  76,  32, 130, 204, 140,  96,
        142, 174, 130, 220, 130, 189, 130, 205,
        131, 116, 131,  64, 131,  67, 131, 139,
        150, 188, 130, 170, 150, 179, 140, 248,
        130, 197, 130, 183, 129,  66,  13,  13,
         10
      ]
    }
    Permission denied, please try again.
    XSL Invalid XSL format (or) file name.

    node:child_process:925
        throw err;
        ^

    Error: Command failed: wmic process where (commandline like "%ssh-askpass.bat%") get processid,parentprocessid /format:"%WINDIR%\System32\wbem\en-us\csv"
    XSL Invalid XSL format (or) file name.



        at checkExecSyncError (node:child_process:848:11)
        at Object.execSync (node:child_process:922:15)
        at Object.<anonymous> (node:electron/js2c/asar_bundle:5:12704)
        at Object.<anonymous> (c:\Users\EDITED_DEADBEAF\.vscode\extensions\ms-vscode-remote.remote-containers-0.266.1\dist\common\sshAskpass.js:5:808)
        at Module._compile (node:internal/modules/cjs/loader:1165:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
        at Module.load (node:internal/modules/cjs/loader:1035:32)
        at Module._load (node:internal/modules/cjs/loader:876:12)
        at c._load (node:electron/js2c/asar_bundle:5:13343)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
      status: 44210,
      signal: null,
      output: [
        null,
        Buffer(0) [Uint8Array] [],
        Buffer(41) [Uint8Array] [
           88,  83,  76,  32, 130, 204, 140,  96,
          142, 174, 130, 220, 130, 189, 130, 205,
          131, 116, 131,  64, 131,  67, 131, 139,
          150, 188, 130, 170, 150, 179, 140, 248,
          130, 197, 130, 183, 129,  66,  13,  13,
           10
        ]
      ],
      pid: 4436,
      stdout: Buffer(0) [Uint8Array] [],
      stderr: Buffer(41) [Uint8Array] [
         88,  83,  76,  32, 130, 204, 140,  96,
        142, 174, 130, 220, 130, 189, 130, 205,
        131, 116, 131,  64, 131,  67, 131, 139,
        150, 188, 130, 170, 150, 179, 140, 248,
        130, 197, 130, 183, 129,  66,  13,  13,
         10
      ]
    }
    ubuntu@192.168.1.100: Permission denied (publickey,password).

    2022-12-08 12:47:03.036 [info] [attempt 1] Invoking resolveAuthority(ssh-remote)
    2022-12-08 12:47:10.480 [error] [Extension Host] (node:13136) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
    (Use `Code --trace-deprecation ...` to show where the warning was created)
    2022-12-08 12:47:10.500 [info] [attempt 1] resolveAuthority(ssh-remote) returned '127.0.0.1:54762' after 7464 ms
    2022-12-08 12:47:13.327 [error] Could not find pty on pty host: CodeExpectedError: Could not find pty on pty host
        at h.J (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:27:6395)
        at h.updateTitle (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:27:2665)
        at Object.call (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:17:5110)
        at r.q (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:15:5390)
        at r.p (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:15:4881)
        at /home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:15:4286
        at n.invoke (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:11:145)
        at C.deliver (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:11:2038)
        at v.fire (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:11:1676)
        at process.O (/home/ubuntu/.vscode-server/bin/5235c6bb189b60b01b1f49062f4ffa42384f8c91/out/vs/platform/terminal/node/ptyHostMain.js:9:20886)
        at process.emit (node:events:526:28)
        at emit (node:internal/child_process:938:14)
        at processTicksAndRejections (node:internal/process/task_queues:84:21)
    2022-12-08 12:47:14.063 [info] [perf] Render performance baseline is 44ms
    2022-12-08 12:47:19.562 [error] [Extension Host] Shell server terminated (code: 255, signal: null)

    XSL Invalid XSL format (or) file name.

    node:child_process:925
        throw err;
        ^

    Error: Command failed: wmic process where (commandline like "%ssh-askpass.bat%") get processid,parentprocessid /format:"%WINDIR%\System32\wbem\en-us\csv"
    XSL Invalid XSL format (or) file name.



        at checkExecSyncError (node:child_process:848:11)
        at Object.execSync (node:child_process:922:15)
        at Object.<anonymous> (node:electron/js2c/asar_bundle:5:12704)
        at Object.<anonymous> (c:\Users\EDITED_DEADBEAF\.vscode\extensions\ms-vscode-remote.remote-containers-0.267.0\dist\common\sshAskpass.js:5:808)
        at Module._compile (node:internal/modules/cjs/loader:1165:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
        at Module.load (node:internal/modules/cjs/loader:1035:32)
        at Module._load (node:internal/modules/cjs/loader:876:12)
        at c._load (node:electron/js2c/asar_bundle:5:13343)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
      status: 44210,
      signal: null,
      output: [
        null,
        Buffer(0) [Uint8Array] [],
        Buffer(41) [Uint8Array] [
           88,  83,  76,  32, 130, 204, 140,  96,
          142, 174, 130, 220, 130, 189, 130, 205,
          131, 116, 131,  64, 131,  67, 131, 139,
          150, 188, 130, 170, 150, 179, 140, 248,
          130, 197, 130, 183, 129,  66,  13,  13,
           10
        ]
      ],
      pid: 10128,
      stdout: Buffer(0) [Uint8Array] [],
      stderr: Buffer(41) [Uint8Array] [
         88,  83,  76,  32, 130, 204, 140,  96,
        142, 174, 130, 220, 130, 189, 130, 205,
        131, 116, 131,  64, 131,  67, 131, 139,
        150, 188, 130, 170, 150, 179, 140, 248,
        130, 197, 130, 183, 129,  66,  13,  13,
         10
      ]
    }
    XSL Invalid XSL format (or) file name.

    node:child_process:925
        throw err;
        ^

    Error: Command failed: wmic process where (commandline like "%ssh-askpass.bat%") get processid,parentprocessid /format:"%WINDIR%\System32\wbem\en-us\csv"
    XSL Invalid XSL format (or) file name.



        at checkExecSyncError (node:child_process:848:11)
        at Object.execSync (node:child_process:922:15)
        at Object.<anonymous> (node:electron/js2c/asar_bundle:5:12704)
        at Object.<anonymous> (c:\Users\EDITED_DEADBEAF\.vscode\extensions\ms-vscode-remote.remote-containers-0.267.0\dist\common\sshAskpass.js:5:808)
        at Module._compile (node:internal/modules/cjs/loader:1165:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
        at Module.load (node:internal/modules/cjs/loader:1035:32)
        at Module._load (node:internal/modules/cjs/loader:876:12)
        at c._load (node:electron/js2c/asar_bundle:5:13343)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
      status: 44210,
      signal: null,
      output: [
        null,
        Buffer(0) [Uint8Array] [],
        Buffer(41) [Uint8Array] [
           88,  83,  76,  32, 130, 204, 140,  96,
          142, 174, 130, 220, 130, 189, 130, 205,
          131, 116, 131,  64, 131,  67, 131, 139,
          150, 188, 130, 170, 150, 179, 140, 248,
          130, 197, 130, 183, 129,  66,  13,  13,
           10
        ]
      ],
      pid: 12652,
      stdout: Buffer(0) [Uint8Array] [],
      stderr: Buffer(41) [Uint8Array] [
         88,  83,  76,  32, 130, 204, 140,  96,
        142, 174, 130, 220, 130, 189, 130, 205,
        131, 116, 131,  64, 131,  67, 131, 139,
        150, 188, 130, 170, 150, 179, 140, 248,
        130, 197, 130, 183, 129,  66,  13,  13,
         10
      ]
    }
    Permission denied, please try again.
    XSL Invalid XSL format (or) file name.

    node:child_process:925
        throw err;
        ^

    Error: Command failed: wmic process where (commandline like "%ssh-askpass.bat%") get processid,parentprocessid /format:"%WINDIR%\System32\wbem\en-us\csv"
    XSL Invalid XSL format (or) file name.



        at checkExecSyncError (node:child_process:848:11)
        at Object.execSync (node:child_process:922:15)
        at Object.<anonymous> (node:electron/js2c/asar_bundle:5:12704)
        at Object.<anonymous> (c:\Users\EDITED_DEADBEAF\.vscode\extensions\ms-vscode-remote.remote-containers-0.267.0\dist\common\sshAskpass.js:5:808)
        at Module._compile (node:internal/modules/cjs/loader:1165:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
        at Module.load (node:internal/modules/cjs/loader:1035:32)
        at Module._load (node:internal/modules/cjs/loader:876:12)
        at c._load (node:electron/js2c/asar_bundle:5:13343)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
      status: 44210,
      signal: null,
      output: [
        null,
        Buffer(0) [Uint8Array] [],
        Buffer(41) [Uint8Array] [
           88,  83,  76,  32, 130, 204, 140,  96,
          142, 174, 130, 220, 130, 189, 130, 205,
          131, 116, 131,  64, 131,  67, 131, 139,
          150, 188, 130, 170, 150, 179, 140, 248,
          130, 197, 130, 183, 129,  66,  13,  13,
           10
        ]
      ],
      pid: 12972,
      stdout: Buffer(0) [Uint8Array] [],
      stderr: Buffer(41) [Uint8Array] [
         88,  83,  76,  32, 130, 204, 140,  96,
        142, 174, 130, 220, 130, 189, 130, 205,
        131, 116, 131,  64, 131,  67, 131, 139,
        150, 188, 130, 170, 150, 179, 140, 248,
        130, 197, 130, 183, 129,  66,  13,  13,
         10
      ]
    }
    Permission denied, please try again.
    XSL Invalid XSL format (or) file name.

    node:child_process:925
        throw err;
        ^

    Error: Command failed: wmic process where (commandline like "%ssh-askpass.bat%") get processid,parentprocessid /format:"%WINDIR%\System32\wbem\en-us\csv"
    XSL Invalid XSL format (or) file name.



        at checkExecSyncError (node:child_process:848:11)
        at Object.execSync (node:child_process:922:15)
        at Object.<anonymous> (node:electron/js2c/asar_bundle:5:12704)
        at Object.<anonymous> (c:\Users\EDITED_DEADBEAF\.vscode\extensions\ms-vscode-remote.remote-containers-0.267.0\dist\common\sshAskpass.js:5:808)
        at Module._compile (node:internal/modules/cjs/loader:1165:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
        at Module.load (node:internal/modules/cjs/loader:1035:32)
        at Module._load (node:internal/modules/cjs/loader:876:12)
        at c._load (node:electron/js2c/asar_bundle:5:13343)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
      status: 44210,
      signal: null,
      output: [
        null,
        Buffer(0) [Uint8Array] [],
        Buffer(41) [Uint8Array] [
           88,  83,  76,  32, 130, 204, 140,  96,
          142, 174, 130, 220, 130, 189, 130, 205,
          131, 116, 131,  64, 131,  67, 131, 139,
          150, 188, 130, 170, 150, 179, 140, 248,
          130, 197, 130, 183, 129,  66,  13,  13,
           10
        ]
      ],
      pid: 10304,
      stdout: Buffer(0) [Uint8Array] [],
      stderr: Buffer(41) [Uint8Array] [
         88,  83,  76,  32, 130, 204, 140,  96,
        142, 174, 130, 220, 130, 189, 130, 205,
        131, 116, 131,  64, 131,  67, 131, 139,
        150, 188, 130, 170, 150, 179, 140, 248,
        130, 197, 130, 183, 129,  66,  13,  13,
         10
      ]
    }
    ubuntu@192.168.1.100: Permission denied (publickey,password).
    ```
    </details>

Steps to Reproduce:

1. Connect to remote Ubuntu machine with Remote SSH extension
2. Connect to Docker container with Dev Containers extension

**Comment by indiaki1o** [comment]
> The workaround I have found is to downgrade Dev Containers v0.262.3.
> 
> Another workaround is to copy `%WINDIR%\System32\wbem\en-JP\csv.xsl` to `%WINDIR%\System32\wbem\en-US\csv.xsl`, but this is not recommended as it would change the system files.

I faced the same issue, and your solution worked. Thank you so much.

**Comment by mawanda-jun** [comment]
I faced the same issue and your solution worked. Thank you very much! However, I think this issue is related to a bug in the latest updates, so if I were you I'd leave this issue opened so they can work on it.

Best.

**Comment by tt4g** [comment]
Unfortunately, it appears that Author does not have the authority to reopen the Issue.

We just don't know the status of the bug fix because the source code is not publicly available, but I believe it will be fixed in the next release.

**Comment by chrmarti** [comment]
A fix is available in Dev Containers v0.268.0-pre-release. Could you give that a try and let me know if that fixes the issue for you? Thanks!

**Comment by mawanda-jun** [comment]
`A fix is available in Dev Containers v0.268.0-pre-release.`
It appears it's working. Now I can successfully log in into my remote container using the password with no hustle.
Thank you and keep going on with the good work!
